static/_redirects

```diff
@@ -454,3 +454,4 @@
 /docs/concepts/overview/what-is-kubernetes/      /docs/concepts/overview/ 301
 /docs/tasks/administer-cluster/verify-signed-images/     /docs/tasks/administer-cluster/verify-signed-artifacts/   301
 
+/kubertenes     /blog/2024/06/06/10-years-of-kubernetes/ 
```

Comment:
Please add the HTTP status code [302](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/302) for redirection, to maintain consistency with the other links in this file. Without this change, it defaults to HTTP 301, which indicates relocation.

```suggestion
/kubertenes     /blog/2024/06/06/10-years-of-kubernetes/ 302
```

Reply:
@dipesh-rawat : review comments are addressed, pls check